.github/workflows/integration_test.yaml

```diff
@@ -20,14 +19,17 @@ jobs:
             value: "--num-units=1"
     uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
     with:
-      setup-devstack-swift: true
-      modules: '["test_core", "test_error", "test_upgrade"]'
       extra-arguments: |
         -m "not (requires_secret)" \
-        --openstack-rc ./openrc \
-        --kube-config ~/.kube/config \
+        --openstack-rc ${GITHUB_WORKSPACE}/openrc \
+        --kube-config ./kube/config \
         --screenshot-dir /tmp \
         ${{ matrix.args.value }}
+      modules: '["test_core", "test_error", "test_upgrade"]'
+      pre-run-script: |
+        -c "sudo microk8s enable registry hostpath-storage"
```

Comment:
Should this also include the step to wait for this to take effect? E.g., similar to this command for container registry: `microk8s kubectl -n container-registry rollout status -w deployment/registry`